website/package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "homepage": "https://ludens-reklamebyra.github.io/react-crisscross",
   "dependencies": {
+    "@ludens-reklame/react-crisscross": "^1.2.1",
```

Comment:
Should this be manually updated or always latest?

Reply:
I think so. If not we have no way of showing of new features in the website.